### DISCUSSION TITLE
Invalid dashboard uid (Battery Health)

### DISCUSSION THREAD

**Discussion by bslime-kr** [teslamate-org/teslamate#3523] [General]
The uid value in my Teslamate's Battery Health dashboard appears to be incorrect.

When checking on hub, the uid of the provided Battery Health dashboard is confirmed like this.

   "title": "Battery Health",
   "uid": "jchmRiqUfXg**TM**",

However, the actual uid value applied to my Teslamate is confirmed like this.

   "title": "Battery Health",
   "uid": "jchmRiqUfXg**Rx**",

As a result, when calling the battery health dashboard from the home screen, the message 'Dashboard not found' appears.
All other dashboards are normal. (only for Battery Health)

How can I Fix it?

**Comment by shanhard** [comment]
I am having the same issue.

**Comment by shanhard** [comment]
I still have this issue. When clicking the battery health dashboard link from teslamate main screen, it loads a 'Dashboard not found' and 'Invalid dashboard UID in annotation request' error. I can still get to the dashboard via grafana menu, but the link from within Teslamate is broken.
